MFVNeutralino/plugins/EventFilter.cc

```diff
@@ -57,6 +69,28 @@ bool MFVEventFilter::filter(edm::Event& event, const edm::EventSetup&) {
   edm::Handle<pat::JetCollection> jets;
   event.getByToken(jets_token, jets);
 
+  const edm::LuminosityBlock& lumi = event.getLuminosityBlock();
+
+  // If pertinent, parse randpar configuration
+  if (parse_randpars) {
+    
+    edm::Handle<GenLumiInfoHeader> gen_header;
+    lumi.getByToken(gen_lumi_header_token, gen_header);
+    
+    std::string rp_config_desc = gen_header->configDescription();
+    std::string str_mass = "MS-" + std::to_string(randpar_mass);
+    std::string str_ctau = "ctauS-" + randpar_ctau;
+    std::string str_dcay = randpar_dcay;
+    std::string comp_string_Zn = "ZH_" + str_dcay + "_ZToLL_MH-125_" + str_mass + "_" + str_ctau + "_TuneCP5_13TeV-powheg-pythia8";
+    std::string comp_string_Wp = "WplusH_" + str_dcay + "_WToLNu_MH-125_" + str_mass + "_" + str_ctau + "_TuneCP5_13TeV-powheg-pythia8";std::string comp_string_Wm = "WminusH_" + str_dcay + "_WToLNu_MH-125_" + str_mass + "_" + str_ctau + "_TuneCP5_13TeV-powheg-pythia8";
+    if (not ((comp_string_Wp == rp_config_desc) or (comp_string_Zn == rp_config_desc) or (comp_string_Wm == rp_config_desc))) {
+      return false;
+    }
+    else {
+      return true;
+    }
+  }
+
```

Comment:
_If_ I understand correctly, `parse_randpars` will only be set true as part of a separate EventFilter from that used for selection events based on the cuts below this, is that right? If so, a comment above this block could be useful as a reminder (otherwise one worries that we exit before applying e.g. our jet HT cuts below!).

Reply:
OH or [this line](https://github.com/DisplacedVertices/cmssw-usercode/pull/7/files#diff-76120c22c7a07aa1ee285ce20bf1f80b30ee2624a7afa698f3c23e8aa30fbd88R113) answers it, in which case we should just add a reminder of that as a comment here as well.